Sources/PCMultipartMessage.swift

```diff
@@ -46,46 +46,134 @@ extension PCMultipartMessage: Hashable {
 }
 
 public struct PCPart {
+    let type: String
```

Comment:
Oh, one more thing. This should be part of the payload according to the spec. (where it was before was correct...)